content/en/docs/concepts/storage/mount-propagation.md

```diff
@@ -0,0 +1,77 @@
+---
+reviewers:
+- saad-ali
+- jsafrane
+- thockin
+- msau42
+title: Mount Propagation
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
```

Comment:
How does this content fit with the updates to the Volumes page?

Reply:
I responded to previous feedback that this content was too narrow/technical to include in the volumes page, and should be moved to a separate task. I'm flexible on where to put it.

However, I would like to keep it in the same PR otherwise it's likely to be dropped completely.